onnxruntime/core/graph/graph.cc

```diff
@@ -2476,24 +2503,38 @@ void Graph::InitFunctionBodyForNode(Node& node) {
         } else
           input_types.emplace_back();
       }
-      onnx::FunctionBodyBuildContextImpl function_body_ctx(node_proto, input_types);
+      ONNX_NAMESPACE::FunctionBodyBuildContextImpl function_body_ctx(node_proto, input_types);
       if (!node.op_->BuildContextDependentFunction(function_body_ctx, onnx_function_proto))
         return;
     } else {
       onnx_function_proto = *(node.op_->GetFunction());
     }
-
-    ORT_TRY {
-      auto func_ptr = std::make_unique<onnxruntime::FunctionImpl>(*this, node.Index(), onnx_function_proto,
-                                                                  logger_);
-      function_container_.emplace_back(std::move(func_ptr));
-      node.SetFunctionBody(*function_container_.back());
-    }
-    ORT_CATCH(const std::exception&) {
-      // Return without using this function op's expansion. No need to fail just yet.
-      // If ORT has a specialized kernel for this op then execution will proceed
+  } else {
+    std::string func_identifier = node.Domain() + ":" + node.OpType();
+    const auto& model_local_functions = this->GetModelLocalFunctions();
+    auto iter = model_local_functions.find(func_identifier);
```

Comment:
nit: we generally don't use the unnecessary `this->` qualifier in our code. is there a reason it's needed here?